video.c

```diff
@@ -183,8 +187,22 @@ video_init(uint8_t *in_chargen, int window_scale, char *quality)
 				       SDL_TEXTUREACCESS_STREAMING,
 				       SCREEN_WIDTH, SCREEN_HEIGHT);
 
+	char icon_path_data[PATH_MAX];
+
+	char *icon_filename = "icon.bmp";
+	char *icon_path = icon_path_data;
```

Comment:
`char *icon_path[PATH_MAX];`

and kill line 190

Reply:
I've removed line 190 however changing line 192 to use ```char *icon_path[PATH_MAX];``` results in an incompatible pointer type error.  I've updated this PR with the other changes suggested however and I'm able to build the emulator with no errors on my Linux system.

Reply:
I've changed icon_filename to a const char array.